CMakeLists.txt

```diff
@@ -43,6 +43,15 @@ message(STATUS "Building sparrow v${${PROJECT_NAME}_VERSION}")
 
 OPTION(BUILD_TESTS "sparrow test suite" OFF)
 
+# Linter options
+# =============
+
+OPTION(ACTIVATE_LINTER "Create targets to run clang-format" OFF)
```

Comment:
IMO it should not be ON by default because you don't want to have as requirement to have clang-format if you just want to compile the project with GCC or MSVC (as in the CI), or as a dependency user (VCPKG, Conan)

Reply:
also if a clang-format exists in a project's source code, msvc will take it into account automatically, vscode too assuming the right extension.